src/cli/index.js

```diff
@@ -77,6 +79,10 @@ inquirer
                     value: TRANSFORMER_CHAI_SHOULD,
                 },
                 {
+                    name: 'Jasmine: this Syntax',
```

Comment:
I'm wondering a bit if we can come up with a more descriptive name... Should we call it "Jasmine: this usage"? Hm.